### PR TITLE
feat(grid): Add grid tests

### PR DESCRIPTION
**Pull request by umahaea**
@JasonLantz @StommePoes @wyseguyonline @udhadpa for your review...
Add grid tests for both Desktop and Mobile.
Cover all the responsive breakpoints.

### CODE REVIEW

**Comment by umahaea** [comment]
@mhomolak

File: src/test/java/elementsTests/GridTest.java [hunk 1]
```diff
@@ -0,0 +1,869 @@
...
+                {"Column11", 480, 800, gridPgObj.cont3Col11, new String[]{"421.656px", "421.667px", "421.65625px"}},
+                {"Column12", 480, 800, gridPgObj.cont3Col12, new String[]{"460px", "462px"}}};
+    }
+
+    @Test(testName = "SM Span Column Width Test", dataProvider = "SM Span Column Width Test Data", groups = {"desktop-regression"})
+    private void smSpanColumnWidthTest(String colName, int width, int height, By element, String[] expectedCSSValue) {
+        chooseEnv();
+        commonUtils.setWindowSize(width, height);
+        colWidth = commonUtils.getCSSValue(element, "width");
+        System.out.println(colWidth);
```
Comment (StommePoes) [inline src/test/java/elementsTests/GridTest.java]: I only see 4 of these printlines... are these leftover from debugging?

File: src/test/java/elementsTests/GridTest.java [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+            isMarginLeft = commonUtils.assertValue(marginRight, expMarginLeft, "The " + rowName + " row margin-right for window size " + width + " is not as per the spec");
+            isMarginRight = commonUtils.assertValue(marginLeft, expMarginRight, "The " + rowName + " row margin-right for window size " + width + " is not as per the spec");
+            Assert.assertTrue(isMarginRight && isMarginLeft);
+        }
+    }
+
+    @DataProvider(name = "XS Span Column Width Test Data")
+    private Object[][] getXSSpanColumnWidthTestData() {
+        return new Object[][]{
+                {"Column1", "width", 479, 800, gridPgObj.cont3Col1, new String[]{"39.0781px", "39.0667px", "39.078125px"}},
```
Comment (StommePoes) [inline src/test/java/elementsTests/GridTest.java]: which 3 browsers was this again? Chrome, Firefox and... IE? You said during the demo but I don't remember.

Comment (umahaea) [inline src/test/java/elementsTests/GridTest.java]: It is Chrome FF and Safari.. For IE/Edge,  setWindowSize() is having some issues, so had to verify that manually.

Comment (StommePoes) [inline src/test/java/elementsTests/GridTest.java]: ok

File: src/test/java/elementsTests/GridTest.java (hunk 1 above)
Comment (umahaea) [inline src/test/java/elementsTests/GridTest.java]: r u talking about this System.out.println(colWidth) mallory? ya, they are left overs from debugging, should have removed it.

Comment (StommePoes) [inline src/test/java/elementsTests/GridTest.java]: ok cool

Comment (umahaea) [inline src/test/java/elementsTests/GridTest.java]: I will remove this.

**Comment by wyseguyonline** [comment]
LGTM

**Comment by JasonLantz** [comment]
looks good....

**Pull request merged**